### DISCUSSION TITLE
Allure report for Dart language

### DISCUSSION THREAD

**Discussion by lravihs** [allure-framework/allure2#2061] [General Discussion]
It will be better if the support for Allure report support is given to dart language

**Comment by lravihs** [comment]
I can only able to feed the test-report.xml file to allure report using **allure serve** command in the terminal

**Comment by baev** [comment]
related issue https://github.com/allure-framework/allure2/issues/1119

**Comment by baev** [comment]
Good news, everyone! Work in progress, the integration will be open sourced next week (or week after that)

**Reply by Moander2243** [reply]
Hi @baev, is there any update about the specific date when this integration will be released?

**Reply by baev** [reply]
I'm stuck a bit with the flutter tests support, but we're getting there

**Reply by baev** [reply]
https://github.com/allure-framework/allure-dart
